protocol-units/bridge/move-modules/sources/atomic_bridge_initiator.move

```diff
@@ -17,6 +17,7 @@ module atomic_bridge::atomic_bridge_initiator {
     const COMPLETED: u8 = 2;
     const REFUNDED: u8 = 3;
 
+    const TIME_LOCK_DURATION: u64 = 24 * 60 * 60; // 24 hours in seconds
```

Comment:
What if you name this INITIATOR_TIME_LOCK_DURATION and do the multiplication by 2 here to get 48 hours? Then you won't have to multiply by 2 anywhere else and you'll have more descriptively named timelock constants.